bin/server.py

```diff
@@ -128,6 +133,7 @@ def __init__(self):
         handlers = [
             (r'/', IndexHandler),
             (r'/termsocket', TermSocketHandler),
+            (r'/control-panel', ControlPanelHandler),
```

Comment:
Substitute `/termsocket` for `/experimental` to show that the feature is not finished.